src/rdo.rs

```diff
@@ -630,11 +630,11 @@ pub fn rdo_mode_decision<T: Pixel>(
             mode_context,
             &mv_stack,
             rdo_type,
-            !needs_rec
```

Comment:
Should this no longer be negated? This one switched from `!needs_rec` to `need_recon_pixel`, but line 637 still has the negation.

Reply:
Yes, because the formal parameter (i.e. old for_rdo_use) in the function encode_block_b() has been changed as well. Please see https://github.com/xiph/rav1e/pull/1284/files/2ec39e87281cd77abebb3ea284fa1692772e8fe1#diff-26006a0497afb96d876927e114a7b29eR1128.